index.js

```diff
@@ -4,27 +4,23 @@ module.exports = function (img, selection_canvas, onCrop) {
   if('function' === typeof selection_canvas)
     onCrop = selection_canvas, selection_canvas = null
 
-  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
-
   onCrop = onCrop || function () {}
 
- var width = img.width, height = img.height
+  var c2 = selection_canvas = selection_canvas || h('canvas.hypercrop__selection', {width: 512, height: 512})
```

Comment:
I just indented this line.
I thought about that change, but dropping c2 would suggest dropping c, and renaming ctx and ctx2. 
ctx is a common name for canvas context, so I think that's good.

TL;DR - I'm inclined to leave this